app/javascript/components/Pages/Borrow.js

```diff
@@ -4,44 +4,45 @@ import { Link } from 'react-router-dom';
 import { Button } from 'reactstrap'
 
 const Borrow = ({ books }) => (
-
     <>
         <h3 className="heading">All Books</h3>
-        { books.map((book, index) => {
-            return (
-                <div key={`book-${index}`} className="bookCard">
-                    <p>{book.title}</p>
-                    <div className="info">
-                        <h4>Author: {book.author}</h4>
-                        <p>Description: {book.description}</p>
-                        <p>Pages: {book.pages}</p>
-                        <p>Series: {book.series}</p>
-                        <p>Pick Up Date: {book.pick_up_date}</p>
-                        <p>Due Date: {book.due_date}</p>
+        {
+            books.map((book, index) => {
+                return (
+                    <div key={`book-${index}`} className="bookCard">
```

Comment:
You can just keep the key as `index` value here

Reply:
lol I think I told him to make it unique :)

Reply:
ha